app/src/main/java/com/example/hris/model/TimeLogsModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.hris.model
+
+import android.os.Parcelable
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.squareup.moshi.JsonClass
+import kotlinx.android.parcel.Parcelize
+
+@JsonClass(generateAdapter = true)
+data class TimeLogsModel(
+    val status: String,
+    val message: String?,
+    val timeLogs: List<TimeLogs>?
+)
```

Comment:
This data class could extend `ResponseModel` to inherit the `status` and `message` fields.